mwafrika/teamwork
# #168283923  create the signup user interface for users

#### What does this PR do?
Create the user signup interface for users to create their accounts
 
#### Description of Task to be completed?
all the input fields are well set for the required information
the responsiveness of the user signup interface is well provided.

#### How should this be manually tested?
Every new user who need to create an account should fill in all the necessary and required information . When a user decrease or increase the size of the website, the system will not loose it performance .

#### Any background context you want to provide?
All input fields are secured and no user can signup with the email that has been taken by another user.

#### What are the relevant pivotal tracker stories?
#168283923 

#### Screenshots (if appropriate)
![signup](https://user-images.githubusercontent.com/25848053/64248698-2fcf8180-cf12-11e9-8ba2-8737f765dc9d.png)
